include/dcgp/expression_ann.hpp

```diff
@@ -949,7 +954,7 @@ class expression_ann : public expression<double>
     d_loss(typename std::vector<std::vector<double>>::const_iterator dfirst,
            typename std::vector<std::vector<double>>::const_iterator dlast,
            typename std::vector<std::vector<double>>::const_iterator lfirst, expression<double>::loss_type loss_e,
-           unsigned parallel) const
+           unsigned parallel = 0u) const
```

Comment:
@darioizzo is it okay to use a default argument in this case? This will prevent users from providing the parallel argument when they want to use it single threaded.

Reply:
sure. Its still weird for users to be able to set it to 3 if the code is compiled with no parallelism ...

Reply:
I updated the documentation to note that the parallel argument has no effect when dcgp is configured with `DCGP_SINGLE_THREAD`.